docs/export-styles.md

```diff
@@ -0,0 +1,54 @@
+# Export Styles
```

Comment:
I am thinking maybe we should just link to https://angular.io/guide/creating-libraries#managing-assets-in-a-library.

Reply:
Yeah, that makes sense.  Probably don't need the link on the README in that case.  Just linking from the Assets doc might be enough?  But it does become a bit circular then.  That page links to the assets doc in ng-packagr, and if we link back to it for the exports portion, that be strange.  What do you think?